packages/apollos-data-connector-postgres/src/features/resolver.js

```diff
@@ -106,9 +106,9 @@ const resolver = {
     id: id('AddCommentFeature'),
   },
   WebviewFeature: {
-    height: ({ data }) => data.height || 400,
+    height: ({ data }) => data?.height || 400,
     id: id('WebviewFeature'),
-    url: ({ data }) => data.url,
+    url: ({ data, url }) => data?.url || url || '',
```

Comment:
```suggestion
    url: ({ data, args }) => data?.url || args.?url || '',
```

like this @camrun91 ?

Reply:
` WebviewFeature: {
    height: ({ args }) => args?.height || 400,
    id: id('WebviewFeature'),
    url: ({ args, url }) => args?.url || url || '',
  },` 

that is what I currently have and it looks to be working. The .css file is not loading in the simulator but I don't think this is related.